apps/crn-server/src/handlers/news/algolia-index-news-handler.ts

```diff
@@ -0,0 +1,79 @@
+import { AlgoliaClient, algoliaSearchClientFactory } from '@asap-hub/algolia';
+import { NewsEvent, NewsResponse } from '@asap-hub/model';
+import { EventBridgeHandler, NewsPayload } from '@asap-hub/server-common';
+import { NotFoundError } from '@asap-hub/errors';
+import { Boom, isBoom } from '@hapi/boom';
+import { EventBridgeEvent } from 'aws-lambda';
+import { algoliaApiKey, algoliaAppId, algoliaIndex } from '../../config';
+import NewsController from '../../controllers/news.controller';
+import { getNewsDataProvider } from '../../dependencies/news.dependencies';
+import logger from '../../utils/logger';
+import { sentryWrapper } from '../../utils/sentry-wrapper';
+import { addTagsFunction } from '../helper';
+
+/* istanbul ignore next */
+export const indexNewsHandler =
+  (
+    newsController: NewsController,
+    algoliaClient: AlgoliaClient<'crn'>,
+  ): EventBridgeHandler<NewsEvent, NewsPayload> =>
+  async (event) => {
+    const eventType = event['detail-type'] as Extract<
+      NewsEvent,
+      'NewsPublished' | 'NewsUnpublished'
+    >;
```

Comment:
Removing this 

```
as Extract<
      NewsEvent,
      'NewsPublished' | 'NewsUnpublished'
    >;
```
it seems that Typescript still understands that `eventType` should be `'NewsPublished' | 'NewsUnpublished'`

![Screenshot 2023-11-29 at 09 30 24](https://github.com/yldio/asap-hub/assets/16595804/e99e6f3e-5a93-4c27-bb92-ad16d836bd24)

So I think we can remove it :)